### PR TITLE
DROOLS-3932: [DMN Designer] Replace BPMN specific parsing error message

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/DROOLS-3932

### CODE REVIEW

**Comment by karreiro** [comment]
Jenkins, please retest this.

**Comment by manstis** [comment]
Hi @karreiro you'll need to run (at least) `drools-wb-weabpp` (or the full workbench).

The DMN standalone editor does not support opening invalid files (the submarine work changes this, but for now, on `master` you'll need to use `drools-wb-webapp` and upload an invalid XML file).

File: kie-wb-common-dmn/kie-wb-common-dmn-project-client/src/main/resources/org/kie/workbench/common/dmn/project/client/resources/i18n/DMNProjectClientConstants.properties [hunk 1]
```diff
@@ -1,3 +1,4 @@
 DMNDiagramResourceType.shortName=DMN
 DMNDiagramResourceType.description=DMN
-DMNDiagramResourceType.download=Download as DMN
\ No newline at end of file
+DMNDiagramResourceType.download=Download as DMN
+DMNDiagramParsingErrorMessage=An error occurred parsing the DMN diagram. The XML content may contain errors.
```
Comment (jomarko) [inline kie-wb-common-dmn/kie-wb-common-dmn-project-client/src/main/resources/org/kie/workbench/common/dmn/project/client/resources/i18n/DMNProjectClientConstants.properties]: @sterobin you have usually perfect ideas about messages shown to the user. Is this one fine for you?

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by manstis** [comment]
@jomarko @karreiro I tried with Business Central locally; and it seemed fine to me.

Screenshot from 2019-04-24 10-19-45

**Comment by manstis** [comment]
Hi @karreiro, I tried again with `drools-wb-webapp` and it worked fine.

Make sure you're comiling all of `kie-wb-common` (or at least `kie-wb-common-stunner` in addition to `kie-wb-common-dmn`).

**Comment by manstis** [comment]
Jenkins please execute full downstream build.

**Pull request merged**